### ISSUE TITLE
encoding of “Guidelines” in Guidelines

### ISSUE THREAD

**Issue by sydb** [TEIC/TEI#2103]
There are over 320 text nodes in the _Guidelines_ that contain the word “Guidelines” (case sensitive). Likely most all of them are references to the _Guidelines_ themselves. A dozen or so of these occurrences are inside `<title>`. Probably either
1. Those dozen should be stripped of their `<title>` status, OR
2. The other ~308 should be encoded in `<title>`.

But I suspect that in either case someone needs to actually look at the occurrences to see what is going on.

Personally, I think of “_Guidelines_” as shorthand for 
~~~xml
<title>
  <choice>
    <abbr>Guidelines</abbr>
   <expan>TEI P5: Guidelines for Electronic Text Encoding and Interchange</expan>
  </choice>
</title>
~~~
so I would be in favor of using `<title>` everywhere it is used as the short title of the _Guidelines_. That said, many of the occurrences are likely to be “these Guidelines”, which should probably be “these guidelines”, not `these <title>Guidelines</title>`, but others may have more intelligent input.

**Comment by JanelleJenstad** [comment]
Practices for citing one's own digital project are inconsistent and still evolving (indeed, I'm still undecided how to talk about MoEML!), which means we have an opportunity to help shape the practices. As Syd points out, "these Guidelines" should be "these guidelines" if we mean the genre of guidelines. If we are referring to _TEI P5: Guidelines for Electronic Text Encoding and Interchange_, we probably have to decide if _Guidelines_ is the short title we want to use, given the potential ambiguity between the short title for our own work and the generic term for the genre. 

I'll do some research, look at the instances in our documents, and come up with a suggestion.

**Comment by JanelleJenstad** [comment]
My strong recommendation is that we wrap _Guidelines_ in the <code>title</code> element when we are referring to _TEI P5: Guidelines for Electronic Text Encoding and Interchange_. Our _Guidelines_ are presented as a book in chapters. The book has an edition and issue number. People cite it in their own bibliographies as a book. We should cite our own book in the same way that we want users to cite it. 

Tasks to do:
Wrap _Guidelines_ in <code>title</code> in the 308 cases where it appears without such tagging.
Add this new practice to our own documentation guidelines. (Do we have documentation guidelines?)

**Comment by lb42** [comment]
Personally, I'd rather see the retention of  "these guidelines" in the many cases where it's not referring to the published work but to its content
e.g. "some versions of these guidelines", "forbidden by these guidelines"
and so on.

**Comment by JanelleJenstad** [comment]
@lb42: Yes, that makes sense. The rule needs to be cognate with how we treat books. So if one would say "this book," or "as this book argues," then one would not capitalize guidelines or wrap it in <code>title</code>. I'll have to go through the 308 cases and decide on a case-by-case basis.

**Comment by martindholmes** [comment]
@lb42 "forbidden by these guidelines" is a very odd phrase, don't you think? If they're guidelines, what business do they have forbidding anything? But a quick search shows the only occurrence is in the phrase "...such practice is not forbidden by these Guidelines, but no recommendations are made...".

**Comment by JanelleJenstad** [comment]
Working on this issue in branch /JanelleJenstad_Issue2013.

**Comment by ebeshero** [comment]
@JanelleJenstad You'll want to create a new branch for this now!  (Old branch had no new commits; we were cleaning up old branches and deleted it.)

**Comment by sydb** [comment]
So just checking (for my work on #2144): 

~~~xml
These guidelines make no recommendation …
~~~
But
~~~xml
Versions of these <title>Guidelines</title> prior to 4.5.0 made no recommendation …
~~~
right?

**Comment by ebeshero** [comment]
Council looked at this today and we agree that @JanelleJenstad should simply implement her proposed use of the `<title>` element for direct references to the TEI Guidelines.
